tests/transforms/test_split_non_commuting.py

```diff
@@ -620,21 +582,23 @@ def circuit():
 )
 
 
+@pytest.mark.parametrize("grouping_strategy", [None, "naive", "pauli", "default"])
 class TestAutodiffSplitNonCommuting:
     """Autodiff tests for all frameworks"""
 
-    @pytest.mark.parametrize("prod", (qml.prod, qml.operation.Tensor))
     @pytest.mark.autograd
-    def test_split_with_autograd(self, prod):
+    def test_split_with_autograd(self, grouping_strategy):
         """Test that results after splitting are still differentiable with autograd"""
+
         dev = qml.device("default.qubit", wires=3)
 
+        @functools.partial(split_non_commuting, grouping_strategy=grouping_strategy)
```

Comment:
Were we just... not using the transform at all and not testing it?

Reply:
Yep 🤣